docs/userguide/objective.md

```diff
@@ -1,24 +1,77 @@
 # Objective
 
-## General Information
+The [`Objective`](baybe.objective.Objective) instructs BayBE how to deal with multiple
+targets (if applicable).
 
-Objectives allows you to effectively manage the optimization objective and are implemented by the [`Objective`](baybe.objective.Objective) class.
-By defining objectives, you can optimize a single target function or combine multiple target functions using different modes.
-To create an objective, it is necessary to provide at least the following:
-* **An optimization mode:** This can be either ``SINGLE`` or ``DESIRABILITY``, denoting either the optimization of a single target function or a combination of different target functions.
-* **A list of targets:** The list of targets that are optimized. Note that the ``SINGLE`` mode also requires a list containing the single target. See [here](./targets) for details.
+To create an objective, it is necessary to provide the following:
+* An optimization `mode`: This can be either `SINGLE` or `DESIRABILITY`,
+  denoting the optimization of a single target function or a combination of
+  different target functions respectively.
+* A list of `targets`: The list of targets that are optimized. Note that the
+  `SINGLE` mode also requires a list containing the single target.
 
-## Supported Optimization Modes
+```{note}
+We are actively working on adding more objective modes for multiple targets.
+```
 
+## Supported Optimization Modes
 Currently, BayBE offers two optimization modes.
 
-### Single Target Optimization: The ``SINGLE`` mode
+### SINGLE
+In the `SINGLE` mode, objectives focus on optimizing a single target. 
+Nearly all of the [examples](../../examples/examples) use this objective mode.
+
+```python
+from baybe.targets import NumericalTarget
+from baybe.objective import Objective
+
+target1 = NumericalTarget(name="yield", mode="MIN", bounds=(0, 100))
+objective = Objective(mode="SINGLE", targets=[target1])
+```
+
+### DESIRABILITY
+The ``DESIRABILITY`` mode enables you to combine multiple targets via scalarization 
+into a single value, a method well know and also utilized in classical DOE.
```

Comment:
"a method well known...." feels like unnecessary boilerplate to me.

Reply:
removed (also had spelling mistake)